frontend/src/component/p-photo-tiles.vue

```diff
@@ -70,6 +71,20 @@
             album: Object,
         },
         methods: {
+            selectRange(photo) {
+                var selection = this.$clipboard.getIds();
+                if (selection.length) {
+                    var lastAddedId = selection[selection.length - 1];
+                    var rangeStart = this.photos.findIndex((photo) => photo.getId() == lastAddedId);
```

Comment:
This might cause issues when you previously selected photos in a different view or in a different search result.